docs/internal.md

```diff
@@ -0,0 +1,71 @@
+# Internal of Azure SignalR Service
+
+- [Server Connections](#server-connections)
+- [Client Connections](#client-connections)
+- [Transport Data between Client and Server](#transport)
+
+Azure SignalR Service is built on top of ASP.NET Core SignalR framework.
+You can start from a local ASP.NET Core SignalR application, then seamlessly move to Azure SignalR Service with a few lines of code change.
+
+The following diagram shows the typical architecture when you use Azure SignalR Service with your own application server.
+Behavior of clients and servers will be different from self-hosting ASP.NET Core SignalR.
+More details are explained in below sections.
+
+![Architecture](./images/arch.png)

```

Comment:
This diagram should show a 2 way connection between the client and service

Reply:
Sure. The arrow is hidden by the client icon. Let me update it.